bfapi/service/jobs.py

```diff
@@ -208,7 +208,7 @@ def create(
         tide_max_24h=tide_max,
     )
 
-def __make_cli_cmd(
+def __make_algorithm_cli_cmd(
```

Comment:
See, now you're just messing with my head. :trollface: 

Can you please describe to me what you think happens when you prefix this particular function with `__`?